app.rb

```diff
@@ -61,6 +61,11 @@ def perform(country_slug, legislature_slug, source = nil)
         end
       end
     end
+
+    with_git_repo.commit_changes_to_branch(branch, "Refresh countries.json") do
+      run('bundle exec rake countries.json')
```

Comment:
This should really run the `EP_COUNTRY_REFRESH=Turkey` (or wherever) version of this, as it'll be much quicker, and will isolate us from conflicts if something else gets merged without updating their country (e.g. all today's PRs that include Bulgaria as well as their own)